selenium.go

```diff
@@ -247,6 +254,12 @@ type WebDriver interface {
 	SendModifier(modifier string, isDown bool) error
 	/* Take a screenshot */
 	Screenshot() ([]byte, error)
+	/* Get the logs configured in the capabilities.
+
+	Common values for log type are 'server', 'browser', 'client', 'driver'.
+	NOTE: will return an error (not implemented) on IE11 or Edge drivers.
+	*/
```

Comment:
I think it might be nicer to enumerate the valid log types.

https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/logging/LogType.html

e.g.
```go
type LogType string

const (
   ServerLog LogType = "server"
   Browser = "browser"
   Client = "client"
   Driver = "driver"
   Performance = "performance"
   Profiler = "profiler"
)
```
(with appropriate documentation, of course).